applications/LinearSolversApplication/custom_solvers/eigen_pardiso_lu_solver.h

```diff
@@ -56,7 +79,10 @@ class EigenPardisoLUSolver
 
     bool Solve(Eigen::Ref<const Vector> b, Eigen::Ref<Vector> x) const
     {
+        const int previous_threads = MKLUtilities::GetNumMKLThreads();
```

Comment:
shouldn't this be the ParallelUtils::GetNumThreads() to be consistent with openmp?